### PR TITLE
New OAuth window

### PR DESCRIPTION
**Pull request by garyluu**
For https://ucsc-cgl.atlassian.net/browse/SEAB-2782 

This now only applies to ORCID account linking

Before PR:
- Click on link account (could be on several different pages)
- Page gets redirected to OAuth screen
- User accepts OAuth stuff
- Page gets redirected to the previous page that the user was on
- Fail message shows if it failed

This PR:
- Click on link account (could be on several different pages)
- A new tiny window pops up
- User accepts OAuth stuff on this new tiny window
- This new tiny window gets redirected to the AuthComponent of Dockstore and does a bit of stuff
  - if it succeeded then automatically close
  - if it failed then the error message would display in that tiny window
- The original page has the userObject refreshed (if the oauth succeeded)

### CODE REVIEW

**Comment by denis-yuen** [comment]
> Kind of a big change. Feels like I'm going backwards and reverting something that was intentional. This affects linking ALL accounts.

 Don't know how I feel about this. We need to change it for ORCID since ORCID has branding guidelines, it doesn't seem necessary for the other ones

**Comment by garyluu** [comment]
> > Kind of a big change. Feels like I'm going backwards and reverting something that was intentional. This affects linking ALL accounts.
> 
> Don't know how I feel about this. We need to change it for ORCID since ORCID has branding guidelines, it doesn't seem necessary for the other ones

Little messier and inconsistent, but I just made it only affect ORCID

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export function assertNoTab(tabName: string): any {
 export function resetDB() {
   before(() => {
     cy.exec('java -jar dockstore-webservice.jar db drop-all --confirm-delete-everything travisci/web.yml');
+    // cy.exec('docker exec -i postgres1 psql webservice_test -U postgres < travisci/db_dump.sql');
```
Comment (denis-yuen) [inline cypress/support/commands.ts:50]: Intentional?

**Review by denis-yuen — APPROVED** [review]
> Little messier and inconsistent, but I just made it only affect ORCID

Will take a look when it makes its way out to dev

File: cypress/support/commands.ts (hunk 1 above)
Comment (garyluu) [reply cypress/support/commands.ts:50]: yes

**Review by coverbeck — APPROVED** [review]

**Review by wshands — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Pull request merged**